src/preloader/mdns.ts

```diff
@@ -19,7 +19,7 @@ export async function mdns4Request(
   return await new Promise((resolve, reject) => {
     const interfaces = allInterfaces();
     if (interfaces.length === 0) {
-      return reject(new Error("No interfaces to send an mDNS request"));
+      reject(new Error("No interfaces to send an mDNS request"));
     }
```

Comment:
To preserve the original behaviour you need to add return to stop the function.
I've just tested it and just `reject` isn't enough to stop the function from executing.

```suggestion
      reject(new Error("No interfaces to send an mDNS request"));
      return;
```

Reply:
Fixed